_plugins/build.rb

```diff
@@ -1,12 +1,9 @@
-Jekyll::Hooks.register :site, :pre_render do |site|
-    # TODO: remove 'js-optimization' before merge
-    production_branches = ['master', 'staging', 'js-optimization']
-    branch = ENV['BRANCH']
+# TODO: remove 'federalist-js' before merge
+production_branches = ['master', 'staging', 'federalist-js']
+branch = ENV['BRANCH']
 
-    if production_branches.include?(branch)
-        puts '[build.rb] webpacking for production'
-        `npm install && npm run webpack > build.log`
-    else
-        puts '[build.rb] not webpacking'
-    end
+if production_branches.include?(branch)
+  puts '[build.rb] webpacking for production'
+  ENV['NODE_ENV'] = 'production'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.